### PR TITLE
tools: update dm-ansible's download description

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? 

some user download latest version, need update dm-ansible's download description

### Which version does your change affect? 

dev, v2.1, v3.0 and v3.1

### CODE REVIEW

File: dev/how-to/deploy/data-migration-with-ansible.md [hunk 1]
```diff
@@ -139,7 +139,8 @@ DM-Ansible 是 PingCAP 基于 [Ansible](https://docs.ansible.com/ansible/latest/
     wget http://download.pingcap.org/dm-ansible-{version}.tar.gz
     ```
 
-    `{version}` 为期望下载的 DM 版本，如 `v1.0.0-alpha`、`latest` 等。
+    `{version}` 为期望下载的 DM 版本，如 `v1.0.1`、`v1.0.2` 等，
```
Comment (csuzhangxc) [inline dev/how-to/deploy/data-migration-with-ansible.md]: How about updating the `latest` used in the sample of this file. like `tar -xzvf dm-ansible-latest.tar.gz &&`.

Comment (WangXiangUSTC) [reply dev/how-to/deploy/data-migration-with-ansible.md]: update in 92c930c

Comment (csuzhangxc) [reply dev/how-to/deploy/data-migration-with-ansible.md]: update `dm_version = latest` too?

Comment (WangXiangUSTC) [reply dev/how-to/deploy/data-migration-with-ansible.md]: update 9fd02b0

**Pull request merged**